cloudtasks/api/TasksSample/TasksSample.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
-
+    <GenerateProgramFile>false</GenerateProgramFile>
```

Comment:
Added based on [this blog post](https://andrewlock.net/fixing-the-error-program-has-more-than-one-entry-point-defined-for-console-apps-containing-xunit-tests/).

Reply:
This has since been reverted